java/vector/src/main/java/org/apache/arrow/vector/BitVector.java

```diff
@@ -578,4 +579,19 @@ public void copyValueSafe(int fromIndex, int toIndex) {
       to.copyFromSafe(fromIndex, toIndex, BitVector.this);
     }
   }
+
+  /**
+   * Create a new vector initialized with the given values.
+   */
+  @VisibleForTesting
+  public static BitVector create(String name, BufferAllocator allocator, int[] values) {
```

Comment:
I think we should use object types here and make it variable length arguments so it can be used as:

create("vectorName", allocator, true, false, null) )i.e. no need to clutter with creating a new array when using the method and the contact is clearer (same goes for other methods)

Reply:
ok, sounds reasonable. Also there is a special case IntervalDayVector takes two values(days and milliSeconds) which is not suitable for this pattern, so keep this vector with arrays?